src/priyanka_Panat/assignment15/BankOperation.java

```diff
@@ -0,0 +1,162 @@
+/*Assignment - 15 : 9th April'2021
+
+	A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+	          a) Person can debit or credit amount from respective bank.
+	              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+	          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+	          c) A Method to show total credit, debit operations 
+	                 output : Total credit operation - 7 times
+	                              Total debit operation - 5 times
+
+	         d) A Method to show individual bank credit operations.
+	                output : ICICI bank Credit Operation - 2
+	                         HDFC bank Credit Operation - 2
+			                 SBI bank Credit Operation - 1
+	                         BOB bank Credit Operation - 4
+
+	         e) A Method to show individual bank debit operations.
+	                output : ICICI bank Debit Operation - 2
+	                         HDFC bank Debit Operation - 2
+			                 SBI
+*/
+
+package priyanka_Panat.assignment15;
+
+
+	public class BankOperation {
+
+			int icicibalance, sbibalance, hdfcbalance,bobbalance;
+			int idebitCount=0,icreditCount=0, sdebitCount=0,screditCount=0, hdebitCount=0, hcreditCount=0,bdebitCount=0,bcreditCount=0;
+			String name;
+			
+			void setBal(int ibal,int sbal,int hbal,int bbal,String custname) {
+				icicibalance=ibal;
+				sbibalance=sbal;
+				hdfcbalance=hbal;
+				bobbalance=bbal;
+				name=custname;
+				System.out.println(name + " Transaction details");
+			}
+			
+			void bankCreditDebitOperation(String bank, String op,int amt) {
+			switch(op) {
+				case "debit":
+				switch(bank) {
+				case "ICICI" :
+						if(icicibalance>=amt) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method